pkg/scalers/azure_app_insights_scaler.go

```diff
@@ -13,6 +13,7 @@ import (
 	"k8s.io/metrics/pkg/apis/external_metrics"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 
+	az "github.com/Azure/go-autorest/autorest/azure"
```

Comment:
Would you mind moving this to the block with generic imports above?

Reply:
Done.